cedar-policy-validator/src/err.rs

```diff
@@ -26,78 +26,82 @@ use thiserror::Error;
 
 #[derive(Debug, Error)]
 pub enum SchemaError {
-    /// Errors loading and parsing schema files
-    #[error("JSON Schema file could not be parsed: {0}")]
-    ParseFileFormat(serde_json::Error),
+    /// Error thrown by the `serde_json` crate during deserialization
+    #[error(transparent)]
```

Comment:
Instead of `transparent`, maybe a little context would be helpful for users, like `"failed to parse schema: {0}"` (the `{0}` part will still display the Serde error)